test/jdk/javax/swing/JPopupMenu/8075063/ContextMenuScrollTest.java

```diff
@@ -46,52 +46,38 @@
 
 public class ContextMenuScrollTest extends JPopupMenu
 {
-    private JMenuItem undo;
-    private JMenuItem redo;
-    private JMenuItem cut;
-    private JMenuItem copy;
-    private JMenuItem paste;
-    private JMenuItem delete;
-    private JMenuItem selectAll;
-    private final Robot robot;
-    private JFrame frame;
-    private JMenuBar menuBar;
-    private JMenu menu;
-    private volatile Point p = null;
-    private volatile Dimension d = null;
+    private static JMenuItem undo;
+    private static JMenuItem redo;
+    private static JMenuItem cut;
+    private static JMenuItem copy;
+    private static JMenuItem paste;
+    private static JMenuItem delete;
+    private static JMenuItem selectAll;
```

Comment:
These can be local variables; as well as the menuBar. They're not accessed anywhere else but createGUI().